src/components/select_device_popup.rs

```diff
@@ -77,6 +93,10 @@ impl Component for SelectDevicePopupComponent {
             KeyCode::Down | KeyCode::Char('j') => self.next()?,
             KeyCode::Enter => self.run_new_app()?,
             KeyCode::Esc => self.hide_popup()?,
+            KeyCode::Char('f') => {
```

Comment:
Can you automatically show select flavors dialog when user click Enter key. If there are no flavors in project, please launch app without showing select flavors modal.